libraries/AP_Airspeed/AP_Airspeed_Backend.h

```diff
@@ -50,6 +50,15 @@ class AP_Airspeed_Backend {
     int8_t get_pin(void) const;
     float get_psi_range(void) const;
     uint8_t get_bus(void) const;
+    bool bus_is_confgured(void) const;
+    uint8_t get_instance(void) const {
+        return instance;
+    }
+
+    // see if voltage correction should be disabled
+    bool disable_voltage_correction(void) const {
+        return (frontend._options.get() & AP_Airspeed::OptionsMask::DISABLE_VOLTAGE_CORRECTION) != 0;
+    }
```

Comment:
```suggestion
    bool voltage_correction_disabled(void) const {
        return (frontend._options.get() & AP_Airspeed::OptionsMask::DISABLE_VOLTAGE_CORRECTION) != 0;
    }
```

Reply:
you want to remove the comment?

Reply:
Yes, and the words have been rearranged; `disable_voltage_correction` is a direction to disable voltage correction, `voltage_correction_disabled` is a query as to the current state IMO.